style/js/shield_defs.js

```diff
@@ -1102,16 +1102,60 @@ export function loadShields(shieldImages) {
       bottom: 5,
     },
   };
-
   shields["US:NY:Thruway"] = {
     norefImage: shieldImages.shield40_us_ny_thruway,
     notext: true,
   };
-
   shields["US:NY:STE"] = {
     norefImage: shieldImages.shield40_us_ny_ste,
     notext: true,
   };
+  [
+    "Albany",
+    "Allegany",
+    "Bronx",
+    "Broome",
+    "Cattaraugus",
+    "Chautauqua",
+    "Chemung",
+    "Chenango",
+    "Clinton",
+    "Columbia",
+    "Delaware",
+    "Dutchess",
+    "Essex",
+    "Franklin",
+    "Fulton",
+    "Greene",
+    "Hamilton",
+    "Herkimer",
+    "Jefferson",
+    "Lewis",
+    "Livingston",
+    "Madison",
+    "Montgomery",
+    "Nassau",
+    "Oneida",
+    "Orange",
+    "Oswego",
+    "Otsego",
+    "Putnam",
+    "Rensselaer",
+    "Rockland",
+    "Saint Lawrence",
+    "Saratoga",
+    "Schoharie",
+    "Schuyler",
+    "Steuben",
+    "Suffolk",
```

Comment:
These routes are currently tagged [`US:NY:CR:Suffolk`](https://taginfo.openstreetmap.org/tags/network=US%3ANY%3ACR%3ASuffolk). Should New York county routes using the blue pentagon include the `:CR:` infix or no?

Reply:
I don't think so, because these route numbers aren't state-assigned. They should probably be retagged to match county roads in the rest of the state.

Reply:
OK, I thought that was the difference between a `:CR` suffix and a `:CR:` infix, but this non-CR format is indeed very common even in states that standardize the sign designs.

Reply:
Hm. I was under the impression that the `:CR:` infix meant the numbering is generally state-coordinated but counties can design their own shields. I guess we should clarify this with the community before supporting one tagging scheme over another.

Reply:
Suffolk County routes have been retagged: https://www.openstreetmap.org/changeset/120479526